tests/lib/cli-engine.js

```diff
@@ -844,6 +844,133 @@ describe("CLIEngine", function() {
             });
         });
 
+        describe("useCache", function() {
+            var sandbox;
+            var fs = require("fs");
+            var rimraf = require("rimraf").sync;
+
+            beforeEach(function() {
+                rimraf(path.resolve(".eslintcache"));
```

Comment:
why not just use `fs` module to do this and remove `rimraf` dependency. We have used fs in lot of other tests also.